app/scripts/store/dimStores.directive.js

```diff
@@ -29,7 +29,7 @@
       scope: {},
       link: Link,
       template: [
-        '<div ng-if="vm.stores.length" ng-class="[\'dim-col-\' + vm.settings.charCol, { \'hide-filtered\': vm.settings.hideFilteredItems, itemQuality: vm.settings.itemQuality }]">',
+        '<div ng-if="vm.stores.length" ng-class="[\'dim-col-\' + vm.settings.charCol, { \'hide-filtered\': vm.settings.hideFilteredItems }]">',
```

Comment:
nice! should we also eventually move the `hide-filtered` class to the index.html file?